app/services/idv/profile_maker.rb

```diff
@@ -48,7 +48,8 @@ def save_profile(
 
     def set_idv_level(in_person_verification_needed:, selfie_check_performed:)
       if in_person_verification_needed
-        if IdentityConfig.store.in_person_proofing_enforce_tmx
+        if IdentityConfig.store.in_person_proofing_enforce_tmx &&
+           FeatureManagement.proofing_device_profiling_decisioning_enabled?
```

Comment:
Do you think we should also update profile_maker_spec.rb to test for this additional logic?

Reply:
That is not a bad idea @gina-yamada

Reply:
@gina-yamada addressed in [bc9d35019bdb79f2c7eec653ef0e86b686f00982](https://github.com/18F/identity-idp/pull/10391/commits/bc9d35019bdb79f2c7eec653ef0e86b686f00982)